wiki/Requeriments_UserStories.md

```diff
@@ -0,0 +1,49 @@
+# Requeriments & User Stories of calculator-angular-tdd
+## Sprint 1
+
+### 1. Funcional requirements 
+
+  1. The user must be able to use a calculator with the functionalities of the operations of multiplying, dividing, adding, subtracting.
+
+  2. The application must support operations between decimals
+  
+  3. Calculator values must be able to be cleaned
+  
+  4. When the value of the calculator is 666 an image of the devil must be displayed
+  
+  5. When the value of the calculator is 42 should be shown to God
```

Comment:
When the value of the calculator is 42 should be shown to God

 When the value of the calculator is 42 a God image must be shown

Reply:
Done